server/athenian/api/controllers/default_controller.py

```diff
@@ -1,21 +1,41 @@
-from typing import List, Dict
+import datetime
+
 from aiohttp import web
+from sqlalchemy import select, sql
 
+from athenian.api import FriendlyJson
+from athenian.api.models import CalculatedMetric, CalculatedMetricValues
 from athenian.api.models.calculated_metrics import CalculatedMetrics
-from athenian.api.models.invalid_request_error import InvalidRequestError
+from athenian.api.models.db import github
+# from athenian.api.models.invalid_request_error import InvalidRequestError
 from athenian.api.models.metrics_request import MetricsRequest
-from athenian.api.models.no_source_data_error import NoSourceDataError
-from athenian.api import util
+# from athenian.api.models.no_source_data_error import NoSourceDataError
 
 
 async def calc_metrics(request: web.Request, body) -> web.Response:
     """Calculate metrics.
 
-    
-
     :param body: Desired metric definitions.
     :type body: dict | bytes
-
     """
     body = MetricsRequest.from_dict(body)
-    return web.Response(status=200)
+    github_repos_query = []
+    for for_set in body._for:
+        for repo in for_set.repositories:
+            if repo.startswith("github.com/"):
```

Comment:
Should it also handle `http(s)://github.com/...`?

Reply:
Sorry, I should have left a comment: this is just a stupid demo, no real logic is hidden.